src/components/Footer/SocialMediaFooterLink.jsx

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright (C) 2018 The Trustees of Indiana University
+SPDX-License-Identifier: BSD-3-Clause
+*/
+import * as React from "react";
+import * as PropTypes from "prop-types";
+
+import * as Rivet from "../util/Rivet";
+
+/**
+ * Use the Social Media Footer Link component to create a link for the social media footer.
+ */
+const SocialMediaFooterLink = ({ children, label, url, ...attrs }) => {
+  if (!url) {
+    return null;
+  }
+  return (
+    <li {...attrs}>
+      <a href={url}>
+        <span className="rvt-sr-only">{label}</span>
```

Comment:
In [this](https://rivet.iu.edu/components/footer/?example=footer-with-social-media-icons-and-resource-links) example, I am seeing that the span can be provided a styling class as well. Did you intentionally omit that since this span is for screen readers only?